lib/ruby_lsp/standard/wraps_built_in_lsp_standardizer.rb

```diff
@@ -0,0 +1,83 @@
+module RubyLsp
+  module Standard
+    class WrapsBuiltinLspStandardizer
+      include RubyLsp::Requests::Support::Formatter
+      def initialize
+        init!
+      end
+
+      def init!
+        @config = ::Standard::BuildsConfig.new.call(ARGV)
```

Comment:
ARGV here is going to be the set of arguments passed to the `ruby-lsp` executable, which are likely not recognized by standard. Could that lead to any issues? Would it be better to pass an empty array every time?

Reply:
The [Logger](/lib/standard/lsp/logger.rb) is careful to only output over stderr, to keep the stdout communication channel clean